### PR TITLE
Telegram Bot: Integrate the retweet feature for admins only

### PR DESCRIPTION
**Pull request by iammarco11**

### CODE REVIEW

**Comment by iammarco11** [comment]
@Aniketh01 if you want to check, just message me on telegram, I will run it locally then you can interact. I have not added your telegram id to the admins list, so you would be seeing a view button when `/start` is sent to the bot.

File: tg_bot.py
Comment (Aniketh01) [inline tg_bot.py]: Does it iterate through the list to check if all the admin ids are checked?

I'm wondering if any() is of use in here?

Comment (Aniketh01) [reply tg_bot.py]: Also, need to check if two users initiate the bot at the same time - does it prevents us from a normal use case or not?

File: tg_bot.py
Comment (Aniketh01) [inline tg_bot.py:54]: For admins we can have both retweet ans view button can be displayed

Comment (iammarco11) [reply tg_bot.py:54]: Okay will add that

File: tg_bot.py
Comment (iammarco11) [reply tg_bot.py]: > Also, need to check if two users initiate the bot at the same time - does it prevents us from a normal use case or not?

I don't think it would be a problem as bots are used at the same time, but if you want to check, we can :)

Comment (Aniketh01) [reply tg_bot.py]: It's better to have it tested than leave it untested

Comment (iammarco11) [reply tg_bot.py]: > Does it iterate through the list to check if all the admin ids are checked? I'm wondering if any() is of use in here?

The if statement just checks whether the user id is there in the list of admin id, if yes it will create the retweet button.

any() will just check whether the list is iterable or not, why would we need to do that?

**Review by Aniketh01 — COMMENTED** [review]
I think we can only merge this work only after  PR #39 gets merged

File: tg_bot.py
Comment (Aniketh01) [inline tg_bot.py]: Need this comment?

**Review by Aniketh01 — CHANGES_REQUESTED** [review]

File: tg_bot.py
Comment (iammarco11) [reply tg_bot.py]: yeah will remove that

**Review by Aniketh01 — APPROVED** [review]

**Pull request merged**
